romfs/source/fighter/koopa/motion/body/motion_patch.yaml

```diff
@@ -50,10 +50,64 @@ attack_air_f:
     cancel_frame: 38
 attack_air_n:
   extra:
-    cancel_frame: 44
+    cancel_frame: 38
```

Comment:
Is this f38 ingame? You motion rated 4f away from the startup, so just making sure

Reply:
Was able to verify this is Frame 38 in game